src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -202,9 +202,9 @@ private FileDataVersionOne CreateFileDataVersionOne(Artifact v2FileData)
                 fileData = new FileDataVersionOne
                 {
                     Hashes = CreateHashVersionOneListFromV2Hashes(v2FileData.Hashes),
-                    Length = v2FileData.Length == -1 ? 0 : v2FileData.Length,
+                    Length = v2FileData.Length == -1 ? 0 : (int)v2FileData.Length,
```

Comment:
>Length = v2FileData.Length == -1 ? 0 : (int)v2FileData.Length, [](http://example.com/codeflow?start=20&length=62)

we will still not be able to convert to v1 without error if the data is too large since we are not changing v1.
So I just use direct cast here.